types/spotify-api/index.d.ts

```diff
@@ -985,6 +982,90 @@ declare namespace SpotifyApi {
         valence: number;
     }
 
+    /**
+     * Audio Analysis Object
+     * https://developer.spotify.com/documentation/web-api/reference/#/operations/get-audio-analysis
+     */
+    interface AudioAnalysisObject {
+        meta: {
+            analyzer_version: string
+            platform: string,
+            detailed_status: string,
+            status_code: number,
+            timestamp: number,
+            analysis_time: number,
+            input_process: string
+        },
```

Comment:
I think these sub-types ought to have their own interfaces, that way it's easier to use only the relevant parts as a argument type for example.

Reply:
Good point. I've extracted the subtypes into their own interfaces. Also, I found out that there is an official prettier configuration for this repo. With the latest commit, the files have been [formatted properly](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).